### PR TITLE
Document unicode

### PR DESCRIPTION
**Pull request by DanGrayson**

### CODE REVIEW

**Comment by benediktahrens** [comment]
Is it possible to do what is described here by shipping the necessary .el files (if license permits) and doing some magic in .dir-locals.el ?

**Comment by DanGrayson** [comment]
It should be.  Shall I try to do it?  The license allows it.

**Comment by benediktahrens** [comment]
On 08/15/2015 06:47 PM, Daniel R. Grayson wrote:
> It should be.  Shall I try to do it?  The license allows it.

I think it would be worth it, since it would make using Unicode
throughout UniMath feasible finally.
I'd be very grateful if you could give it a try.

**Comment by DanGrayson** [comment]
Okay, try it out.

File: emacs/agda/LICENSE [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+--------------------------------------------------------------------------------
+
+The file src/full/Agda/Utils/ReadP.hs is Copyright (c) The University of
```
Comment (benediktahrens) [inline emacs/agda/LICENSE]: The second half of this file can be removed, since it concerns a file we do not ship.

**Comment by benediktahrens** [comment]
I have checked, it works well for me. Thanks a lot!

**Comment by DanGrayson** [comment]
My recent commit takes your suggestion in to account.

**Pull request merged**